libs/milvus/tests/integration_tests/vectorstores/test_milvus.py

```diff
@@ -441,6 +442,106 @@ def test_milvus_vector_field(temp_milvus_db: Any) -> None:
     assert_docs_equal_without_pk(output, [Document(page_content="foo")])
 
 
+def test_milvus_multi_vector_embeddings() -> None:
+    sparse_embedding_func = BM25SparseEmbedding(corpus=fake_texts)
+    dense_embedding_func_1 = FakeEmbeddings()
+    dense_embeddings_func_2 = FakeEmbeddings()
+    docsearch = Milvus.from_texts(
+        embedding=[
+            sparse_embedding_func,
+            dense_embedding_func_1,
+            dense_embeddings_func_2,
+        ],
+        texts=fake_texts,
+        connection_args={"uri": "./milvus_demo.db"},
```

Comment:
Can this uri be passed in the same way as ut above using temp_milvus_db? Just for the code consistency